packages/office-addin-lint/src/commands.ts

```diff
@@ -23,7 +23,11 @@ export async function lint(command: commander.Command) {
     const pathToFiles: string = getPathToFiles(command);
     performLintCheck(pathToFiles);
   } catch (err) {
-    process.exitCode = err && typeof err.status == "number" ? err.status : defaults.ESLintExitCode.CommandFailed;
+    let outputExitCode = err && typeof err.status == "number" ? err.status : defaults.ESLintExitCode.CommandFailed;
```

Comment:
I think you could change this a better way. I don't think adding the `Unexpected error detected.` message helps (but I might need to see it to make sure I understand what the output is when the command has an unexpected error).  The key thing here is that after catching it so the exit code can be set, the error should be retrhown so the error message is shown (by the top-level catch handler).